packages/teleterm/src/ui/services/keyboardShortcuts/useKeyboardShortcut.ts

```diff
@@ -0,0 +1,29 @@
+import { useEffect } from '@gravitational/shared/hooks';
+import { useAppContext } from 'teleterm/ui/appContextProvider';
+import {
+  KeyboardShortcutEvent,
+  KeyboardShortcutEventSubscriber,
+  KeyboardShortcutType,
+} from './types';
+
+export function useKeyboardShortcut(
+  handler:
+    | ((shortcutEvent: KeyboardShortcutEvent) => void)
+    | Partial<Record<KeyboardShortcutType, () => void>>
```

Comment:
Lets simply the API by allowing to subscribe only to specific bindings (without wildcards). 
```ts
type Handlers = Partial<Record<KeyboardShortcutType, () => void>>

export function useKeyboardShortcut(handlers: Handlers){

}
```